flake.nix

```
@@ -1,42 +1,189 @@
 {
   inputs = {
-    nixpkgs.follows = "std/nixpkgs";
-    std.url = github:divnix/std;
-    tullia = {
-      url = github:input-output-hk/tullia;
-      inputs.std.follows = "std";
-    };
+    haskellNix.url = "github:input-output-hk/haskell.nix";
```

maybe worth sharing `tullia`?

```suggestion
    haskellNix.url = "github:input-output-hk/haskell.nix";
    haskellNix.inputs.tullia.follows = "tullia";
```

---

I guess. thought we could also use tullia from haskell.nix?